tests/testthat/test-hard-arrange.R

```diff
@@ -0,0 +1,41 @@
+context("test-arrange")
+
+setup({
+  
+  df = disk.frame:::gen_datatable_synthetic(1e3+11)
+  data.table::fwrite(df, "tmp_pls_delete_gb.csv")
```

Comment:
I have moved all of these files to `tempdir()` in the `development` branch. You may want to merge with `development` branch and do a PR to `development` branch

Reply:
Updated PR to development branch.